webapp/index.js

```diff
@@ -0,0 +1,13 @@
+import app from "./app.js"
+import products from "./routes/products"
+import reviews from "./routes/reviews"
+import users from "./routes/users"
+
```

Comment:
As you're describing here the routing middleware for your app, this code should be exported to `webapp/app.js` and then applied along with other middlewares. The code should look like here
```
// imports

export default function(app) {
  app.use('/api', products); // GET and POST
  ...and so on
}
```

and in webapp/app.js you should do

```
import router from './routes'
...
const app = express();
app.use(router(app));
...
```

And yeah, please move this file to `routes/index.js` as it is a root file for your routing middleware